build-dotnet/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+# Config
+
+dnf install -y clang python-unversioned-command
+
+git clone https://github.com/raulsntos/godot-dotnet
+cd godot-dotnet
+git checkout upgrade-assistant-plus-source-code-plugin-wip
```

Comment:
Currently it's using the latest commit on the WIP branch.

We'll have to decide whether we want to have a local Git checkout and set the commit manually (like we do for Godot itself), or always build the latest dev branch. I'm not sure how the versioning for godot-dotnet will be like long term.